index.html

```diff
@@ -3,12 +3,13 @@
 
  <head>
   <title>Tic Tac Toe</title>
-
+  <link type="text/css" rel="stylesheet" href="style.css">
  </head>
 
  <body>
   <p>Hello World! You are grey and sleepy today.</p>
   <p>Terminal doesn't work as expected (under my command)! Hovever at least I created package.json and changed license...</p>
+  <div id="board"></div>
```

Comment:
Correct thinking but I didn't ask you for that. Nothing to do with the task. Remove.